stories/Molecules/Blocks/BioCard/biocard.scss

```diff
@@ -0,0 +1,161 @@
+@import "../../../assets/scss/variables";
+@import "../../../assets/scss/mixins";
+@import "../../../assets/scss/base";
+
+/* feature card start */
+.bio-card {
+  background: $color-gray-200;
+  border-bottom: 0;
+  height: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: column-reverse;
```

Comment:
@harishkumar-srijan if you want to add flex-direction for small device then you can use the media query for small device and can remove the overwrite code written for medium device